default.yml

```diff
@@ -43,7 +43,7 @@ org_settings:
     metadata: $FLEET_SSO_METADATA
   integrations:
   mdm:
-    apple_bm_default_team: Workstations
+    apple_bm_default_team: $FLEET_APPLE_BM_DEFAULT_TEAM
```

Comment:
```suggestion
```

Hey @getvictor maybe we remove it from the example so there's less secrets to worry about for new users? 

We still want to support it.

The new "GitOps" reference page in the docs will serve as the reference for all supported options.

Reply:
Removed.